UsbAccess/UsbAccess.cpp

```diff
@@ -216,12 +274,55 @@ namespace Plugin {
         bool result = false;
 
         std::list<string> paths;
+        string baseURL = LINK_URL_HTTP;
+
+		string pathParam;
+		if (parameters.HasLabel("path"))
+			pathParam = parameters["path"].String();
+
+		bool bLinkExists = false;
+
+        //Loop through all the existing IDs and set bLinkExists to true, if it already has the link
+        for(auto itr : m_CreatedLinkIds)
+        {
+            string linkPath = LINK_PATH+std::to_string(itr);
+			bLinkExists = isSymlinkExists(linkPath, pathParam);
+			if(bLinkExists)
```

Comment:
If link already exists, log suitable message and return that link. It shouldn't error out.

Reply:
Fixed in pr: https://github.com/rdkcentral/rdkservices/pull/3673